maykin_common/accounts/csrf.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.http import HttpResponseRedirect
+from django.views.csrf import (
+    CSRF_FAILURE_TEMPLATE_NAME,
+    csrf_failure as original_csrf_failure,
+)
+
+
+def csrf_failure(request, reason="", template_name=CSRF_FAILURE_TEMPLATE_NAME):
+    """
+    Catch CSRF failure when tryin to login a second time, when already logged
+    in, by redirecting to the LOGIN_REDIRECT_URL.
+    """
+    if request.path in settings.LOGIN_URLS and request.user.is_authenticated:
```

Comment:
`LOGIN_URLS` is not a standard Django setting, so this should go through the `maykin_common.settings.get_setting` machinery.